src/container/plugins/PausePlugin.js

```diff
@@ -13,10 +13,12 @@
 	{
 		/**
 		 * Reference to the pause application button
-		 * @property {jquery} pauseButton
+		 * @property {HTMLElement} pauseButton
 		 */
-		this.pauseButton = $(this.options.pauseButton)
-			.click(onPauseToggle.bind(this));
+		this.pauseButton = document.querySelectorAll(this.options.pauseButton);
+
+		this.pauseButton[0].addEventListener('click', onPauseToggle.bind(this));
```

Comment:
Why are we hard coding to two elements here? Should we make this a loop? On that note, maybe all of those `querySelector` calls should be `querySelectorAll` calls? I think the general intent when instantiating a container is that the selector should match a single item (for `pauseButton`, `soundButton` etc). This would _technically_ break backwards compatibility since we're only going to match the first.

Reply:
Not I fully agree with the querySelectorAll idea, besides the pause button is this something that frequently happens? If that is the case then It makes sense, but I don't really agree with the idea otherwise.